### ISSUE TITLE
A DateField with Type="Month" collapses on click inside a picker in Chrome

### ISSUE THREAD

**Issue by DaniilVeriga** [extnet/Ext.NET#668]
Test case

    <ext:DateField runat="server" Type="Month" />

This bug report appears to be related:
http://www.sencha.com/forum/showthread.php?289825

Fixed in the revision 6292 (trunk). It goes to 3.1.0 beta.

Review after next ExtJS upgrade.

**UPDATE**: still reproducible with ExtJS 5.1.1.

**Comment by DaniilVeriga** [comment]
[This forum thread](http://forums.ext.net/showthread.php?59749) reports that it is still an issue if use a DateField as a GridPanel Column's Editor. Corrected the fix in the revision 6553 (trunk). It should be in time to go to 3.2.1.

**Comment by DaniilVeriga** [comment]
One more correction according to [this forum post](http://forums.ext.net/showthread.php?59749&p=274331&viewfull=1#post274331). Applied in the revision #6549 (trunk). It goes to 3.3.

**Comment by fabriciomurta** [comment]
**Update:** Sencha issue _EXTJS-14607_ allegedly fixed in ExtJS 5.1.0.47 release.

**Comment by DaniilVeriga** [comment]
EXTJS-14607 is related to the bug stated in this Issue, but it is not the same problem. Yes, the ExtJS bug has been fixed, but it doesn't fix this Issue. The Issue here is similar, but it only happens with Ext.NET, because Ext.NET introduces `Type="Month"` setting which is not available in pure ExtJS. `Type="Month"` setting uses a MonthPicker in a way that MonthPicker is not designed in ExtJS to be used. MonthPicker is only designed to be opened from a DatePicker, but it is not designed to substitute a DatePicker. I don't consider this bug a Sencha one anymore.

As well, I found that the issue is reproducible in Firefox and the fix had to be corrected to fix it in Firefox as well. Along with that the initial fix has been optimized.
